README.md

```diff
@@ -22,4 +22,18 @@ children = root.children # returns all child nodes
 ```
 
 ## Documentation
-For documentation, run `make html` in the `docs` directory.
+1. Install sphinx and sphinx_rtd_theme
+```shell
+pip install sphinx sphinx_rtd_theme
```

Comment:
these are in the dev deps

Reply:
> these are in the dev deps

I included them because I needed them to make the docs. I might attach my terminal output to show what I mean.

Reply:
TLDR: I was not able to make html until I installed shinx and sphinx_rtd_theme.